jobs/base.bull.job.js

```diff
@@ -11,6 +11,10 @@ class BaseJob {
 		this.logger = log.createLogger({ group: this.constructor.name, jobName: this.constructor.name, queueName });
 		this.queueName = queueName;
 		this.concurrency = concurrency;
+		const lockDuration = overrideQueueOptions && overrideQueueOptions.settings
+			? overrideQueueOptions.settings.lockDuration
```

Comment:
this could result in an `undefined` `lockDuration`, no? maybe default it similar to `lockRenewTime` below?

Reply:
It is being defaulted to 60000. If the lockDuration param is not passed in overrideQueueOptions, then it will default it to 60 seconds